posthog/api/person.py

```diff
@@ -95,7 +96,10 @@ def _filter_request(self, request: request.Request, queryset: QuerySet) -> Query
             contains = []
             for part in parts:
                 if ":" in part:
-                    queryset = queryset.filter(properties__has_key=part.split(":")[1])
+                    matcher, key = part.split(":")
+                    if matcher == "has":
```

Comment:
Context on this change?

Reply:
It's just to make sure that we only treat the has_key behaviour if the search term is actually `has:X`, to avoid weird edge cases on trying to use different filters that we do not support yet.